--- /phase1/fixtures/prsdd-go5-diagnostic/go3pr2-priming.in	2026-08-05 21:19:54.806558050 +0000
+++ /phase1/fixtures/prsdd-go5-diagnostic/go5pr-database-list.in	2026-08-05 21:19:55.012117303 +0000
@@ -2,28 +2,22 @@
 Q
 Q
 go2
-CM12
-Q
-go5
-100 -200
+SM95
+go60
 1
+300 320
+2
 DSG
+D
 Q
-go3
+go5
+go13
+300 320
 2
-A
-90 1 90
-E
-1000
-Q
-Q
+year
+1950 2050
 DSG
-995 1005
-Q
-PLOT
-Q
 GO99
 C
 Y
- 
 qt
